docs/cadt_rpc_api.md

```diff
@@ -1628,64 +1983,342 @@ Functionality: Show the complete history of an organization
+Using the above `curl` will save the results to a file in the current directory called `project-offer.txt`.
+
+
+#### Get the details of the currently uploaded offer file
+
+- Only fetches purposed changes, makes no changes
+- Used by the offer taker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+Request
+```shell
+curl --location --request GET 'localhost:31310/v1/offer/accept' --header 'Content-Type: application/json' > project-offer.txt
+```
+
+Response
+
+```json
+TODO
+```
+
+
+### POST Examples
+
+#### Upload an offer file
+
+- Upload and parse an offer file named `offer-file.txt` in the current directory
+- Used by the offer maker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+```shell
+curl --location --request POST 'localhost:31310/accept/import' --header 'Content-Type: multipart/form-data' \
+--form 'file=./offer-file.txt'
+```
+
+Response
+
+```json
+TODO
+```
+
+### DELETE Examples
+
+#### Cancel the currently active offer
+
+- Used by the offer maker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+Request
+```shell
+curl --location --request DELETE 'localhost:31310/v1/offer/accept/cancel' --header 'Content-Type: application/json'
+```
+
+Response
+
+```json
+{
+  "message": "Active offer has been canceled",
+  "success": true
+}
+```
+
+#### Reject the currently imported transfer offer file
+
+- Used by the offer taker
+- For additional information please see [A Note On CADT Offers](#a-note-on-cadt-offers)
+
+Request
+```shell
+curl --location --request DELETE 'localhost:31310/v1/offer/accept/cancel' --header 'Content-Type: application/json'
+```
+
+Response
+
+```json
+{
+  "message": "Offer Cancelled",
+  "success": true
+}
+```
+
+### Additional offer resources
+
+* POST `offer/accept/commit` - (offer maker) accept and commit the currently uploaded offer file
+
+
+## `governance`
+
+Functionality: create a governance node and manage governance data
+
+### GET Examples
+
+#### Get the UID's of all organizations registered in governance data
+
+Request
+```shell
+curl --location --request GET 'localhost:31310/v1/governance/meta/orgList' --header 'Content-Type: application/json'
+```
+
+Response
+```json
+[
+  {
+    "orgUid": "a9d374baa8ced8b7a4add2a23f35f430fd7a3c99d1480d762e0b40572db4b024"
+  },
+  {
+    "orgUid": "fa47700cb693529602c3eab47a5d681ffe0145dabeee6c69cabdd7869537b917"
+  },
+  {
+    "orgUid": "6cde6a6e4e997952ca01500d830904a084267e2390008d2ae5ca46ed549373ef"
+  },
+  {
+    "orgUid": "5fe508f320f32f51cc53c337d4d1bab5b7a87f8aaeb53fdbdaa87f57eef74b5b"
+  },
+  {
+    "orgUid": "e3a5854f359a6dece0bd822a0f79342be4552da0d6ab21b89085599f3ca0fa45"
+  },
+  {
+    "orgUid": "b3d4e71d806e86ff1f8712b6854d65e2c178e873ee22b2f7d0da937dacbaa985"
+  },
+  {
+    "orgUid": "aca536f31e0d9e9b25c311fa452763282e7ee21d2be684483a293923ca9ab012"
+  }
+]
+```
+
```

Comment:
Can we add a GET governance pickList here? Picklist is really what the audience cares about. Also, could we put it above the orgList GET request?